File: app/src/main/java/com/goblinvault/BankTransferActivity.java
package com.goblinvault;
import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

public class BankTransferActivity extends AppCompatActivity {

    @SuppressLint("MissingInflatedId")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bank_transfer);

        Button transferButton = findViewById(R.id.btnTransfer_transfer);
        transferButton.setOnClickListener(view -> {
            Intent intent = new Intent(this, MainActivity.class);
            startActivity(intent);
        });
    }

    public void setTitle(String title) {
        TextView titleTextView = findViewById(R.id.tvTransfer);
        titleTextView.setText(title);
    }
}

File: app/src/main/java/com/seedling/demo/hostdeveloperdemo/BankTransferActivity.kt
package com.seedling.demo.hostdeveloperdemo

import android.annotation.SuppressLint
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class BankTransferActivity : AppCompatActivity() {

    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_bank_transfer)

        findViewById<Button>(R.id.btnTransfer_transfer).setOnClickListener{
            val intent = Intent(this, MainActivity::class.java)
            startActivity(intent)
        }
    }

    fun setTitle(title: String) {
        findViewById<TextView>(R.id.tvTransfer).text = title
    }
}